### PR TITLE
Enhance OS detection for sysv startup with Debian 8.

### PR DESCRIPTION
**Pull request by mkofahl**
On Debian 8 the file `/etc/lsb-release` does not exist. The `lsb_release` command may be used instead.

### CODE REVIEW

File: opendj-packages/resources/sysv/opendj [hunk 1]
```diff
@@ -45,7 +45,7 @@ elif [ -f /etc/SuSE-release ] ; then
     # SuSE
     . /etc/rc.status
     LOCKFILE=/var/run/rcopendj
-elif [ -f /etc/lsb-release ] ; then
+elif [ -f /etc/lsb-release ] || lsb_release -a >/dev/null 2>&1 ; then
     # Debian
```
Comment (JnRouvignac) [inline opendj-packages/resources/sysv/opendj]: Please add a comment on the line after this one:

```bash
# Debian
# On Debian 8 the file /etc/lsb-release does not exist. The lsb_release command may be used instead.
```

**Comment by JnRouvignac** [comment]
Hello,

Thanks for your contribution!
I have a few comments that you may want to consider before I can merge your change.
I will make the comments inline in the commit.

Note that our official repository is at https://stash.forgerock.org/projects/OPENDJ/repos/opendj/
Although I can merge this change for you on such a simple commit.
For future contributions, please consider using the officiel repository.

**Comment by JnRouvignac** [comment]
Do you want to attribute copyright for this change?

&nbsp;

You may attribute copyright to the organization on behalf of which you are making the change, or you may attribute it to yourself if you have the right to.
You may also decide to not assign copyright. We do not require copyright attribution for a one-liner.

&nbsp;

To attribute copyright, please update `opendj-packages/resources/sysv/opendj` and add a line below ForgeRock's copyright like this:

```bash
# Copyright 2013-2015 ForgeRock AS.
# Portions Copyright 2016 The Organization
```

Or:

```bash
# Copyright 2013-2015 ForgeRock AS.
# Portions Copyright 2016 Martin Kofahl
```

Thanks!

**Comment by mkofahl** [comment]
Thanks for merging and your explanations @JnRouvignac. No copyright required in this case.

**Comment by JnRouvignac** [comment]
I created [OPENDJ-3132](https://bugster.forgerock.org/jira/browse/OPENDJ-3132) out of this PR.
And I merged the commit in master: See [stash](https://stash.forgerock.org/projects/OPENDJ/repos/opendj/commits/317f75527a25f7c8a13824088723d515d9c59b68) and github's 317f75527a25f7c8a13824088723d515d9c59b68

**Comment by JnRouvignac** [comment]
Thank you so much!
Please close the PR if you're happy with the resolution.

**Comment by BrunoLavitForgerock** [comment]
I've closed the PR as the fix has been merged in the original OpenDJ repository (Bitbucket).

**Pull request closed without merging**